ci-operator/step-registry/upi/install/azure/upi-install-azure-commands.sh

```diff
@@ -74,6 +74,14 @@ SSH_PUB_KEY=$(<"${CLUSTER_PROFILE_DIR}/ssh-publickey")
 export CLUSTER_NAME
 export BASE_DOMAIN
 
+provisioned_vnet_file="${SHARED_DIR}/customer_vnet_subnets.yaml"
+if [ -f "${provisioned_vnet_file}" ]; then
+    echo "vnet already created"
+    vnet_name=$(python3 -c "import yaml;data = yaml.full_load(open('${provisioned_vnet_file}'));print(data['platform']['azure']['virtualNetwork'])")
```

Comment:
Is there a way to make getting the variables from this file easier? Typically the file laid down in the shared dir should just have variables ready to export. Using python seems a bit overkill, but I'm not familiar with this file.

Reply:
If not use python, then have to install `yq` to get the wanted key from the shared yaml file, but I saw there are already several places using python to load the wanted key-value, e.g:
```
CLUSTER_NAME=$(python3 -c 'import yaml;data = yaml.full_load(open("install-config.yaml"));print(data["metadata"]["name"])')
```
So here I am just following the same way.

Reply:
Ok this seems reasonble. provisioned_vnet_file looks like it contains stanzas of the install config so I see now why you need to use python.

Reply:
Yes, you are right.